src/app/employee-form/employee-form.component.ts

```diff
@@ -17,15 +18,38 @@ export class EmployeeFormComponent implements OnInit {
     public appConfig = AppConfig;
     loading: boolean = false;
     formPath: any;
+    formio: any;
+    public form: any;
+    currentUser: any;
 
-    constructor(private authService: AuthService, private http: Http, private activatedRoute: ActivatedRoute, private auth: FormioAuthService) { }
+    constructor(private authService: AuthService, private http: Http, private activatedRoute: ActivatedRoute, private auth: FormioAuthService) {
+        this.activatedRoute.queryParams.subscribe(params => {
+            this.formPath = params['formPath'];
+            this.formio = new Formio(this.appConfig.appUrl + '/' + this.formPath);
+        });
+    }
 
     ngOnInit() {
+        this.formio.loadForm().then(form => (this.form = form));
         this.auth.ready.then(() => {
+            this.currentUser = Formio.currentUser();
             this.showEmployeeScreen = this.authService.showEmployeeScreen;
-            this.activatedRoute.queryParams.subscribe(params => {
-                this.formPath = params['formPath'];
-            });
+        });
+    }
+
+    // onSubmit(event) {
+    //     const eventI = event;
+    //     eventI.data.employeeId = Formio.currentUser().__zone_symbol__value._id;
+    //     this.formio.saveSubmission(eventI).then(function (created) {
+    //         console.log("Hello5 :: ", created);
+    //     });
+    // }
+
+    onSubmit(event) {
+        const eventI = event;
+        eventI.data.employeeId = Formio.currentUser().__zone_symbol__value;
+        this.formio.saveSubmission(eventI).then(function (created) {
+           console.log("Hello5 :: ", created);
```

Comment:
@mayurmamoria This will log the id, created-date or the whole object?

Reply:
@vikasbo This will save the whole object.

Reply:
@mayurmamoria The question is on `console.log("Hello5 :: ", created);`

Reply:
@vikasbo Sorry. My bad. It will give back response. I think I have to put code to go to next page. I think it will be employee home screen. Is it correct navigation after submitting the form?

Reply:
Correct, but that can be a separate feature branch. And there, you can remove console.log if it could leak data.